app/controllers/api/v1/user/words_controller.rb

```diff
@@ -16,6 +16,20 @@ def create
     end
   end
 
+  def show
+    word = Word.find(params[:id])
+    books = Glossary.where('word_id = ?', word.id).pluck(:book_id) #need to also add user_id, passed as a param
+    books.map do |book|
+      # create a method in the bookservice to lookup the volume, grab title and id
+      # BookService.volume_lookup (https://www.googleapis.com/books/v1/volumes/volumeId)
+      # make book POROS
+    end
```

Comment:
Since we set the book id to be the volume id when we the poros for the search it should be easy enough to call book service for a single book a few times. If you look at the show user books serializers they should be storing the g_id. There is already a book facade/poro and a book with shelves facade/poro if either of those are helpful.